Source/cmake/GStreamerDependencies.cmake

```diff
@@ -1,3 +1,4 @@
 WEBKIT_OPTION_DEPEND(USE_GSTREAMER_MPEGTS ENABLE_VIDEO)
 WEBKIT_OPTION_DEPEND(USE_GSTREAMER_TRANSCODER ENABLE_MEDIA_RECORDER)
 WEBKIT_OPTION_DEPEND(USE_GSTREAMER_WEBRTC ENABLE_WEB_RTC)
+WEBKIT_OPTION_DEPEND(USE_FLITE ENABLE_SPEECH_SYNTHESIS)
```

Comment:
We're missing the
WEBKIT_OPTION_DEPEND(USE_SPIEL ENABLE_SPEECH_SYNTHESIS)
to ensure that USE_SPIEL is disabled if speech synthesis is disabled.